AEPMessaging/Sources/Messaging+PublicAPI.swift

```diff
@@ -20,17 +20,15 @@ import UserNotifications
     ///   - response: UNNotificationResponse object which contains the payload and xdm informations.
     ///   - applicationOpened: Boolean values denoting whether the application was opened when notification was clicked
     ///   - customActionId: String value of the custom action (e.g button id on the notification) which was clicked.
-    /// - Returns: boolean value that signifies whether the notification originated from AJO and whether the API has proceeded with notification tracking.
     @available(*, deprecated, message: "This method is deprecated. Use Messaging.handleNotificationResponse(:) instead to automatically track application open and handle notification actions.")
     @objc(handleNotificationResponse:applicationOpened:withCustomActionId:)
-    @discardableResult
-    static func handleNotificationResponse(_ response: UNNotificationResponse, applicationOpened: Bool, customActionId: String?) -> Bool {
+    static func handleNotificationResponse(_ response: UNNotificationResponse, applicationOpened: Bool, customActionId: String?) {
```

Comment:
did this change ever get released?  seems like the old version that returns a bool made it into main already

Reply:
No, this change got into the main branch https://github.com/adobe/aepsdk-messaging-ios/pull/208 
But we never released 4.1.0 and pivoted towards using an enum instead of boolean